include/nbl/video/utilities/CComputeBlit.h

```diff
@@ -4,6 +4,7 @@
 
 namespace nbl::video
 {
+template <bool UseGLSL = true>
```

Comment:
don't modify the existing class, rename this one `CComputeBlitGLSL` and copy all the code to `CComputeBlit`

this is because we'll eventually delete the GLSL version and only keep the HLSL one

Reply:
I'm telling you to do this because in the HLSL version we can get rid of silly things like `#include "nbl/builtin/glsl/blit/parameters.glsl"` in the middle of this class, and just include it at the top so parameters are `nbl::hlsl::blit::parameters_t` and not `nbl::video::CComputeBlit<>::parameters_t` in C++

plus a whole bunchof other braindead patterns GLSL forced us into

Reply:
for example the 
```cpp
template <typename BlitUtilities>
	inline void buildParameters
```
method can move to become a `static parameters_t create` pseudo-constructor defined directly in the HLSL